runs/comparisons/suprb_tuning.py

```diff
@@ -81,49 +83,91 @@ def suprb_ES_GA_space(trial: Trial, params: Bunch):
             'rule_generation__mutation__sigma', *sigma_space)
         params.rule_generation__init__fitness__alpha = trial.suggest_float(
             'rule_generation__init__fitness__alpha', 0.01, 0.2)
-        #params.rule_generation__operator = trial.suggest_categorical(
+        # params.rule_generation__operator = trial.suggest_categorical(
         #    'rule_generation__operator', ['&', ',', '+'])
 
-        #if params.rule_generation__operator == '&':
+        # if params.rule_generation__operator == '&':
         #    params.rule_generation__delay = trial.suggest_int('rule_generation__delay', 10, 100)
 
-        #params.rule_generation__mutation = trial.suggest_categorical(
+        # params.rule_generation__mutation = trial.suggest_categorical(
         #    'mutation', ['Normal', 'HalfnormIncrease', 'UniformIncrease'])
-        #params.rule_generation__mutation = getattr(mutation, params.rule_generation__mutation)()
+        # params.rule_generation__mutation = getattr(mutation, params.rule_generation__mutation)()
 
         # GA
         params.solution_composition__selection = trial.suggest_categorical(
```

Comment:
Given past tuning results, can we choose one of these?

Reply:
Tournament is usually the best. I change it from tuning to a fix default parameter